tokens/rounded.json

```
@@ -1,22 +1,27 @@
 {
   "rounded": {
     "none": {
+      "$description": "None - No radius.",
       "value": "0",
       "themeable": true
     },
     "smallest": {
+      "$description": "Smallest available radius discernible from 0. Use this value for rounded elements inside elements that have {rounded.small} applied as border-radius.",
       "value": "2px",
       "themeable": true
     },
     "small": {
+      "$description": "Small - The standard for ui elements that shall appear rounded. Use this value for form elements and other smaller ui elements. When in doubt, pick this one.",
       "value": "3px",
       "themeable": true
     },
     "medium": {
+      "$description": "Medium - The \"next best\" radius. Use for larger organisms like list items or areas, that should appear softened but not aggressively rounded.",
       "value": "4px",
       "themeable": true
     },
     "large": {
+      "$description": "Large - The biggest radius. Use for boxes that purposefully should appear rounded - plays nice with, rounded elements inside it that are rounded via $dp-token-rounded-small.",
```

As we want to reference the new token syntax, this would be

```suggestion
      "$description": "Large - The biggest radius. Use for boxes that purposefully should appear rounded - plays nice with, rounded elements inside it that are rounded via {rounded.small}.",
```